changelog.readme

```diff
@@ -1,3 +1,6 @@
+## 1.3.0
```

Comment:
1.3.0 is activating refund, so bump to 1.4.0?
```suggestion
## 1.4.0
```

Reply:
Yes :)